autorest/codegen/serializers/builder_serializer.py

```diff
@@ -546,6 +546,9 @@ def pop_kwargs_from_signature(self, builder: OperationType) -> List[str]:
                 kwargs_to_pop, ParameterLocation.QUERY
             )
             else PopKwargType.SIMPLE,
+            # maily for api_version of operation to use value set in _config(it does not apply for multiapi).
+            enable_config=bool(kwargs_to_pop)
+            and not self.code_model.options["multiapi"],
```

Comment:
it is necessary now for current "multiapi" since `_config` is from combined Configuration with no `api_version` defined instead of Configuration of subfolder